samples/nrf9160/nrf_cloud_mqtt_multi_service/src/application.c

```diff
@@ -23,42 +26,51 @@ LOG_MODULE_REGISTER(application, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
 /* Timer used to time the sensor sampling rate. */
 static K_TIMER_DEFINE(sensor_sample_timer, NULL, NULL);
 
+/* AT command request error handling */
+#define AT_CMD_REQUEST_ERR_FORMAT "Error while processing AT command request: %d"
+#define AT_CMD_REQUEST_ERR_MAX_LEN sizeof(AT_CMD_REQUEST_ERR_FORMAT) + 17
```

Comment:
why 17?

Reply:
It's `ceil(log_10(2^64)) - 2 + 1`

I don't think anything actually guarantees that int is a 32-bit type, though I could be mistaken

Reply:
ok

Reply:
I'll just go with 20, shaving off a few bytes is probably not worthwhile, especially since I am assuming the potential for 64-bit error values